pipeline/console/container/__init__.py

```diff
@@ -260,6 +260,26 @@ def _build_container(namespace: Namespace):
     #     _print(f"Created tag {pipeline_repo}:{pipeline_tag}", "SUCCESS")
 
 
+def _check_user_has_run_container_up():
+    user_input = (
+        input(
+            "Have you checked your pipeline works locally by running 'pipeline container up'? (yes/no): "
+        )
+        .strip()
+        .lower()
+    )
+    if user_input == "yes":
+        return
+    elif user_input == "no":
+        _print(
+            "Please run the command 'pipeline container up' before continuing. This will help ensure your pipeline works as expected before uploading it to Mystic."
+        )
+        sys.exit()
+    else:
+        _print("Invalid input. Please answer 'yes' or 'no'.")
```

Comment:
Maybe just showing the prompt is enough without making them type out yes? Also think we should include a command line option to skip the prompt

Reply:
yeah, I would agree. I was wondering if there's a way we can detect whether they've ran `pipeline container up` and then displaying a warning if they haven't. (Only way I can really think of doing it would be creating a temporary file or saving some kind of marker somewhere)

Reply:
Cool thanks for the comments, have updated accordingly.
@rossgray Yeah I did wonder about that but considered that getting something simple out to begin with would do.